src/main/java/org/torproject/onionoo/util/DomainNameSystem.java

```diff
@@ -0,0 +1,62 @@
+package org.torproject.onionoo.util;
```

Comment:
License blurb is missing.

Reply:
Added this.

Reply:
I usually only include the current year in newly created files, but I'm also fine with including several years. Why 2013 though? (Mainly for future reference.)

Reply:
Ah ok. Usually I've used copyright years on a per-project basis, not per-file. Will change to just 2018 for consistency.